packages/node/src/plugins/operator/OperatorPlugin.ts

```diff
@@ -70,6 +71,8 @@ export interface OperatorServiceConfig {
     getEthersOverrides: () => Promise<Overrides>
 }
 
+const STAKED_OPERATORS_CACHE_MAX_AGE = 7 * 24 * 60 * 60 * 1000
```

Comment:
Maybe we can make it a bit shorter to play it safe.
```suggestion
const STAKED_OPERATORS_CACHE_MAX_AGE = 2 * 24 * 60 * 60 * 1000
```